mrtt-ui/src/styles/forms.js

```diff
@@ -1,16 +1,29 @@
+import { Typography } from '@mui/material'
 import { styled } from '@mui/material/styles'
 
-export const MainFormDiv = styled('div')(() => ({
+export const MainFormDiv = styled('div')`
   display: 'flex',
-  flexDirection: 'column',
-  justifyContent: 'center',
-  alignItems: 'center',
-  margin: '1.5em'
-}))
+  flex-direction: column;
+  justify-content: center;
+  align-items: stretch;
+  padding: 1.5em;
+  width: 100%;
+  box-sizing: border-box;
```

Comment:
I think this was to make sure the error text wasnt on the same line as the submit button.